test/dlc_tests/sanity/test_binary_visibility.py

```diff
@@ -16,6 +16,8 @@ def test_binary_visibility(image: str):
     'https://' may still be private, codebuild 'build' job uses 'curl' i.e. unsigned request to fetch them and hence should
     fail if an 'https://' link is still private
     """
+    if is_trcomp_image(image):
+        pytest.skip("Skipping test for HF TrComp images due to labels from base image being carried over")
```

Comment:
Please create an Issue that we can track to re-enable this for trcomp DLCs.

Reply:
done